### PR TITLE
Added a new card by g-s01

### PR DESCRIPTION
**Pull request by g-s01**

### CODE REVIEW

**Review by Preet-Sojitra — APPROVED** [review]
LGTM
Your PR will be merged soon.

**Review by nzaih1999 — APPROVED** [review]
Hello @g-s01 your changes are ready to merge. Your card will be live in a few minutes. Happy coding

File: index.html [hunk 1]
```diff
@@ -115,6 +115,48 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+
+        <!-- ________ Gautam's card START ________  -->
+        <div class="card">
+          <p class="name">Gautam Sharma</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="" target="_blank">@g-s01</a>
```
Comment (TenTraicion) [inline index.html:124]:
```suggestion
            <a href="https://github.com/g-s01" target="_blank">@g-s01</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@g-s01, Your Card looks good, but you've forgot to add the link to your user profile. Kindly do the change and commit, It will be merged soon! Thanks

**Pull request closed without merging**